website/react-magma-docs/src/pages/api/form.mdx

```diff
@@ -29,23 +29,128 @@ import {
 } from 'react-magma-dom';
 
 export function Example() {
+  const [state, setState] = React.useState({
```

Comment:
For some reason I am not seeing this example in the docs page? 🤔 
![image](https://github.com/user-attachments/assets/ec2c8d8d-0abf-49b1-8b6c-71c198a3e2c5)